### PR TITLE
Adds support for MapSequence animator via animations from pyvistaqt

### PR DESCRIPTION
**Pull request by jeffreypaul15**
Animations are relatively simple to perform, we could create a separate animator class which wraps `BackgroundPlotter` from `pyvistaqt` for future animations but as of now I think this will suffice.
- [ ] Test has to be added

Wokring :
```py
from sunpy.data.sample import AIA_193_IMAGE, AIA_171_IMAGE
from sunpy.map import Map
from sunkit_pyvista import SunpyPlotter

m = Map(AIA_193_IMAGE)
m2 = Map(AIA_171_IMAGE)

plotter = SunpyPlotter()
plotter.plot_map_sequence(m,m2)
```

image

### CODE REVIEW

File: setup.cfg [hunk 1]
```diff
@@ -34,6 +34,8 @@ include_package_data = True
 setup_requires =
   setuptools_scm
 install_requires =
+  pyqt5
+  pyvistaqt>=0.5.0
```
Comment (nabobalis) [inline setup.cfg:38]: Might be worth making these optional?

File: sunkit_pyvista/mapsequence_animator.py [hunk 2]
```diff
@@ -0,0 +1,27 @@
+import time
+
+
+class SequenceAnimator(object):
+    def __init__(self, time, map_meshes, color_maps):
+        self.angle = 0
+        self.animation_state = False
+        self.start_time = 0
+        self.time = time
```
Comment (nabobalis) [inline sunkit_pyvista/mapsequence_animator.py:9]: What is the time module used for?

File: sunkit_pyvista/mapsequence_animator.py [hunk 3]
```diff
@@ -0,0 +1,27 @@
+import time
```
Comment (nabobalis) [inline sunkit_pyvista/mapsequence_animator.py:1]: Should the file be called `mapsequence_animator.py`?

File: setup.cfg (hunk 1 above)
Comment (jeffreypaul15) [reply setup.cfg:38]: Yeah, I was going to suggest that

File: sunkit_pyvista/mapsequence_animator.py (hunk 2 above)
Comment (jeffreypaul15) [reply sunkit_pyvista/mapsequence_animator.py:9]: time interval for each map to be displayed

File: sunkit_pyvista/mapsequence_animator.py (hunk 3 above)
Comment (jeffreypaul15) [reply sunkit_pyvista/mapsequence_animator.py:1]: I'm not sure, I have created an animator which specifically performs only this function

**Comment by jeffreypaul15** [comment]
On a side note, there seems to be an issue with pytest and pyvistaq.
The code doesn't run with pytest and I'm not sure why

**Comment by nabobalis** [comment]
It will need debugging.

**Comment by jeffreypaul15** [comment]
> It will need debugging.

Yeah, I'll look into it

**Comment by jeffreypaul15** [comment]
Adding `DISPLAY` as an env variable solves the issue and the tests pass locally although they fail on Azure dev-ops, I'm not sure why this is the case

**Comment by jeffreypaul15** [comment]
I shall move this to another class, and open another PR.

**Pull request closed without merging**